x-pack/plugins/observability/public/components/shared/exploratory_view/series_editor/columns/data_type_select.tsx

```diff
@@ -57,6 +57,10 @@ export function DataTypesSelect({ seriesId, series }: Props) {
       if (reportType === ReportTypes.CORE_WEB_VITAL) {
         return id === DataTypes.UX;
       }
+      // NOTE: Logs does not have a config for Performance Distribution
+      if (reportType === ReportTypes.DISTRIBUTION && id === DataTypes.LOGS) {
+        return false;
+      }
```

Comment:
This works :+1:

`<nit-pick>`
The way this is phrased, if anyone adds a report type that is not handled by one of the early `return`s above, this would incorrectly fall through to `true`. How about phrasing this as an allow-list?

```suggestion
      // NOTE: Logs only provides a config for KPI over time
      if (id === DataTypes.LOGS) {
        return reportType === ReportTypes.KPI;
      }
```
`</nit-pick>` :innocent: